### DISCUSSION TITLE
Сильно тормозит на телефонах, не возможно пользоваться v3.8.2

### DISCUSSION THREAD

**Discussion by nexoos-team** [cdek-it/widget#33] [General]
Добрый день, пожалуйста читайте без эмоций)
Сами разработчики и знаем как звучат замечания от клиентов.

1) Версия 3.8.2 очень тормозит на телефонах, просто зависает и не возможно пользоваться.
Касается телефонов на андройде, на айфоне немного быстрее, но тоже тормозит.

2) Если использовать опцию fixBounds: 'locality' становится лучше.
Но появляются следующие баги:
1) при перемещении карты к другому городу, пункты выдачи не появляются.
2) при указании defaultLocation например Казань, вообще не появляются никакие пункты выдачи.

3) На андройде если масштабировать карту двумя пальцами, то увеличивается все и не возможно вернуть обратно.

**Comment by vermorag** [comment]
Добрый день
1. Обсуждается в #34 Сейчас работаем над версией 4.0. В ней отказались от общего протокола 2.0 и реализовали отдельно методы для виджета.
3. Учтем в 4 виджете. В 3 технически реализуется с трудом из-за использования геокодера яндекса и в нем локации не всегда совпадают с наименованиями в нашей базе
4. С ограничением карты или без?
5. Можно запись экрана? Очень похоже на общий баг карты, тк масштабирование хэндлится именно на стороне яндекса, но посмотрим можно ли на это как-то повлиять. Буду благодарен, если запись будет сразу с учетом последних версий виджета.
